games-emulation/mame/mame-0.210.recipe

```diff
@@ -0,0 +1,224 @@
+	lib:libSDL2_ttf$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libutf8proc$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	glm${secondaryArchSuffix}_devel
+	devel:libexpat$secondaryArchSuffix
+	devel:libflac$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:liblua$secondaryArchSuffix >= 5.3
+	devel:libogg$secondaryArchSuffix
+	devel:libpugixml$secondaryArchSuffix
+	devel:librapidjson$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libSDL2_ttf$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libutf8proc$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python2
+	cmd:sed
+	cmd:strip$secondaryArchSuffix
+"
+
+defineDebugInfoPackage mame$secondaryArchSuffix \
+	$commandBinDir/castool \
+	$commandBinDir/chdman \
+	$commandBinDir/floptool \
+	$commandBinDir/imgtool \
+	$commandBinDir/jedutil \
+	$commandBinDir/ldresample \
+	$commandBinDir/ldverify \
+	$commandBinDir/mame$bits \
+	$commandBinDir/nltool \
+	$commandBinDir/nlwav \
+	$commandBinDir/pngcmp \
+	$commandBinDir/regrep \
+	$commandBinDir/romcmp \
+	$commandBinDir/split \
+	$commandBinDir/src2html \
+	$commandBinDir/srcclean \
+	$commandBinDir/testkeys \
+	$commandBinDir/unidasm
+
+BUILD()
+{
+#	make clean
+
+	# This is wrong on so many levels
+	SYS_LDFLAGS="-lbsd -lnetwork " # trailing space required!
+	for _i in flac libutf8proc lua RapidJSON sqlite3; do
+		SYS_CFLAGS+="`pkg-config $_i --cflags`"
+		SYS_LDFLAGS+="`pkg-config $_i --libs`"
+	done
+
+	ARCHOPTS="$SYS_CFLAGS $SYS_LDFLAGS" \
+	make \
+		REGENIE=1 \
+		TOOLS=1 \
+		PRECOMPILE=0 \
+		NOWERROR=1 \
+		USE_SYSTEM_LIB_EXPAT=1 \
+		USE_SYSTEM_LIB_ZLIB=1 \
+		USE_SYSTEM_LIB_JPEG=1 \
+		USE_SYSTEM_LIB_FLAC=1 \
+		USE_SYSTEM_LIB_LUA=1 \
+		USE_SYSTEM_LIB_SQLITE3=1 \
+		USE_SYSTEM_LIB_UTF8PROC=1 \
+		USE_SYSTEM_LIB_GLM=1 \
+		USE_SYSTEM_LIB_RAPIDJSON=1 \
+		USE_SYSTEM_LIB_PUGIXML=1 \
+		$jobArgs
+}
+
+INSTALL()
+{
+	# Based on https://projects.archlinux.de/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/mame
+	# If you change any path here, you will have to adjust it in mame.sh.in too!
+
+	# Arch cleanup
+	# 32 bit main binary called as "mame" which would collide with our symlink
+	if [ "$targetArchitecture" = x86_gcc2 ]
+	then
+		cp mame mame$bits
+	fi
+
+	# Install the binaries
+	mkdir -p "$commandBinDir"
+	for _i in castool chdman floptool imgtool jedutil ldresample ldverify \
+	mame$bits nltool nlwav pngcmp regrep romcmp split src2html srcclean \
+	testkeys unidasm; do
+		install -m755 $_i -t "$commandBinDir"
+		strip "$commandBinDir"/$_i
+	done
+	ln -s "$commandBinDir/mame$bits" "$commandBinDir/mame"
```

Comment:
Why is this necessary? Does `mame` produces `mame64` and `mame32` on both arch?

If not, I think we'll be better off only having a `mame` executable.